assets/src/styles/main.scss

```diff
@@ -1,12 +1,14 @@
 @import 'leaflet';
 
 @import './uswds';
+@import '../../../node_modules/font-awesome/scss/font-awesome';
 
 $usgs-blue: #02274B;
 
-@import './tooltip';
-@import './header';
-@import './footer';
+@import './partials/footer';
```

Comment:
Do we really need a separate directory for the partials? Seems redundant with the file naming with an underscore.

Reply:
Not strictly needed, of course... just seemed useful to distinguish pages/component/partials in that manner. I'm inclined to keep it like that but don't feel too strongly about it.

Reply:
I also don't feel strongly about this.